src/route/AppRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import { Route, Routes } from 'react-router-dom';
+
+import { Layout } from '../components/Layout/Layout';
+import EmployeesPage from '../pages/EmployeesPage/EmployeesPage';
+import LoginPage from '../pages/LoginPage/LoginPage';
+import SignupPage from '../pages/SignupPage/SignupPage';
+
+export default function AppRoute() {
+  return (
+    <Routes>
+      <Route path="/" element={<Layout />}>
+        <Route path="/signup" element={<SignupPage />} />
+        <Route path="/login" element={<LoginPage />} />
+        <Route path="/employees" element={<EmployeesPage />} />
+        <Route path="*" element={<LoginPage />} />
+      </Route>
+    </Routes>
+  );
+}
```

Comment:
![image](https://user-images.githubusercontent.com/108259301/215051070-83266902-df14-4561-9108-26e0de2b2ce8.png)
No form here. The user can be redirected automatically